stable/ambassador/values.yaml

```diff
@@ -3,7 +3,7 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
-replicaCount: 3
+replicaCount: 1
```

Comment:
Not listed as a breaking change :)

Reply:
This is always a mistake. Thank you for finding it